cmd/parca-agent/main.go

```diff
@@ -296,24 +306,170 @@ func run(logger log.Logger, reg *prometheus.Registry, flags flags) error {
 		}
 		if r.URL.Path == "/" {
 			w.Header().Set("Content-Type", "text/html; charset=utf-8")
-			statusPage := template.StatusPage{}
+			statusPage := template.StatusPage{
+				ProfilingInterval:   flags.ProfilingDuration,
+				ProfileLinksEnabled: flags.LocalStoreDirectory == "",
```

Comment:
Let's introduce a variable for this state. `profileLinksEnabled` or `localStorageEnabled` and use this variable for the rest of the code rather than checking the flag equivance to an empty string. It would be easier to make sense of.